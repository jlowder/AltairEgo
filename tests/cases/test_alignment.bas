10 PRINT "     S       M       T       W       T       F       S"
20 PRINT
30 FOR I = 1 TO 7
40 PRINT I;TAB(4+8*I);
50 NEXT I
60 PRINT
