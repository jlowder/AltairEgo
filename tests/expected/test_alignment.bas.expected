Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
     S       M       T       W       T       F       S

 1          2       3       4       5       6       7
OK
